pkg/apis/networking/validation/validation.go

```diff
@@ -277,26 +282,24 @@ func validateHTTPIngressRuleValue(httpIngressRuleValue *networking.HTTPIngressRu
 	if len(httpIngressRuleValue.Paths) == 0 {
 		allErrs = append(allErrs, field.Required(fldPath.Child("paths"), ""))
 	}
-	for i, rule := range httpIngressRuleValue.Paths {
-		if len(rule.Path) > 0 {
-			if !strings.HasPrefix(rule.Path, "/") {
-				allErrs = append(allErrs, field.Invalid(fldPath.Child("paths").Index(i).Child("path"), rule.Path, "must be an absolute path"))
+	for i, path := range httpIngressRuleValue.Paths {
+		idxPath := fldPath.Child("paths").Index(i).Child("path")
+		if !strings.HasPrefix(path.Path, "/") {
+			allErrs = append(allErrs, field.Invalid(idxPath, path.Path, "must be an absolute path"))
+		}
+
+		if path.PathType != nil {
+			if !supportedPathTypes.Has(string(*path.PathType)) {
+				allErrs = append(allErrs, field.NotSupported(fldPath.Child("paths").Index(i).Child("pathType"), path.PathType, supportedPathTypes.List()))
```

Comment:
...then this would be `idxPath.Child("pathType")`

Reply:
Updated, thanks!